packages/contractkit/src/identity/metadata.ts

```diff
@@ -139,11 +139,19 @@ export class IdentityMetadataWrapper {
           throw new Error("Can't claim self")
         }
         break
-      case ClaimTypes.DOMAIN:
+      case ClaimTypes.DOMAIN: {
         const existingClaims = this.data.claims.filter((el: any) => el.domain === claim.domain)
         if (existingClaims.length > 0) {
           return existingClaims[0]
         }
+        break
+      }
+      case ClaimTypes.KEYBASE: {
```

Comment:
One should be able to claim multiple keybase accounts, right?

Reply:
The idea here is to have only claim for each username, similarly to the domains above.

Reply:
Ah right my bad